dev-client/src/model/soilId/soilIdPlaceholders.ts

```diff
@@ -97,3 +105,5 @@ export const DATA_BASED_SOIL_MATCH: DataBasedSoilMatch = {
   dataMatch: SOIL_MATCH_INFO,
   combinedMatch: SOIL_MATCH_INFO,
 };
+
+export const SOIL_DATA: any = {};
```

Comment:
This is a placeholder to be replaced with actual API integration in [#653](https://github.com/techmatters/terraso-product/issues/653).